internal/db/sqlite/folderdb_update.go

```diff
@@ -251,6 +251,10 @@ func (s *folderDB) DropAllFiles(device protocol.DeviceID) error {
 
 	// Recalc global for the entire folder
 
+	if err := s.dropIndexIDLocked(txp, deviceIdx); err != nil {
+		return wrap(err)
+	}
+
 	if err := s.recalcGlobalForFolder(txp); err != nil {
```

Comment:
nit:  
Comment should likely stay next to `recalcGlobalForFolder`? Though frankly, I'd just drop it - doesn't add any meaningful additional information compard to the method name.